site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -215,67 +208,82 @@ export const WorkspaceScheduleForm: FC<WorkspaceScheduleFormProps> = ({
       <form onSubmit={form.handleSubmit} className={styles.form}>
         <Stack>
           {submitScheduleError && <ErrorSummary error={submitScheduleError} />}
-          <TextField
-            {...formHelpers("startTime", Language.startTimeHelperText)}
-            disabled={isLoading}
-            InputLabelProps={{
-              shrink: true,
-            }}
-            label={Language.startTimeLabel}
-            type="time"
-          />
+          <Section title="Start">
```

Comment:
On most of our content containers, we have a border. I noticed we left one off here - was this by-design?

Reply:
Also, should the titles for these sections be translated?

Reply:
Yes, totally forgot to come back and pull out the strings. And the border was an oversight too!